canvasapi/canvas.py

```diff
@@ -552,6 +553,29 @@ def get_calendar_events(self, **kwargs):
             _kwargs=combine_kwargs(**kwargs),
         )
 
+    def get_comm_messages(self, user_id, **kwargs):
+        """
+        Retrieve a paginated list of messages sent to a user.
+
+        :calls: `GET /api/v1/comm_messages \
+        <https://canvas.instructure.com/doc/api/comm_messages.html#method.comm_messages_api.index>`_
+
+        :returns: Paginated list containing messages sent to user
+        :rtype: :class:`canvasapi.paginated_list.PaginatedList` of
+            :class:`canvasapi.comm_message.CommMessage`
+
+        """
+
+        kwargs["user_id"] = user_id
```

Comment:
Taking in `user_id` works well here, but to improve this we can leverage `obj_or_id` to accept either a `User` object or a str/int